doc/user-guide/source/cli_access_instance_through_a_console.rst

```diff
@@ -36,17 +28,5 @@ The command returns a URL from which you can access your instance:
    +--------+------------------------------------------------------------------------------+
    | Type   | Url                                                                          |
    +--------+------------------------------------------------------------------------------+
-   | xvpvnc | http://192.168.5.96:6081/console?token=c83ae3a3-15c4-4890-8d45-aefb494a8d6c  |
+   | novnc  | http://192.168.5.96:6081/console?token=c83ae3a3-15c4-4890-8d45-aefb494a8d6c  |
```

Comment:
That IP looks like it might be from a devstack setup?  ME cluster outputs URLs that look like this if you want a closer representation of a real url:

| novnc | https://vncproxy.dreamcompute.com:6080/vnc_auto.html?token=43b0a98a-fb44-423b-a6c6-723c6f67745a |